transforms/units.js

```diff
@@ -78,7 +78,7 @@ module.exports = (file, api) => {
     const { quasis, expressions } = quasi;
     // Substitute all ${interpolations} with arbitrary test that we can find later
     // This is so we can shove it in postCSS
-    const substitutionNames = expressions.map((value, index) => `__${index}substitution__`);
+    const substitutionNames = expressions.map((value, index) => `/*__${index}substitution__*/`);
```

Comment:
I think comments are stripped from `decl.value`. Does this work for something that adds units (i.e. `top: 50` or `flex: ${const} 0 0`)?

There's also the somewhat undocumented `decl.raws.value.raw` that keeps comments in, so that might work!

Reply:
@jacobp100 Can you give me some test cases, including inputs and expected outputs?

Reply:
###### Input

```js
// Doesn't add units to interpolated values in properties that need units
styled.View`
  top: ${top};
  padding: 0 ${horizontal};
  margin: ${top} 0 ${bottom} 0;
`;
```

###### output
```js
// Doesn't add units to interpolated values in properties that need units
styled.View`
  top: ${top};
  padding: 0px ;
  margin: ${top} 0px  0px;
`;
```

It looks like the first comment in a deceleration is stored in `decl.raws.between`, but the rest are stored in `decl.raws.value.raw`, which is why it worked most of the time.